packages/react/src/composite/TimePicker/src/components/TimePickerAmPm/TimePickerAmPm.tsx

```
@@ -0,0 +1,105 @@
+  );
+
+  return (
+    <div
+      className={classnames.ampmContainer}
+      role="radiogroup"
+      aria-label={selectorLabel}
+    >
+      <div className={classnames.amButtonState[current ?? "default"]}>
+        <TimePickerOption
+          ref={amButtonRef}
+          role="radio"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<input type="radio">

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)